src/gui/transferlistwidget.h

```diff
@@ -34,6 +34,7 @@
 #include <QTreeView>
 
 #include "base/bittorrent/infohash.h"
+#include "transferlistmodel.h"
```

Comment:
I don't know what `Q_DECLARE_METATYPE` does but if there is no way to avoid this include then there is a corresponding forward declaration below (for removal).

Reply:
Qt need to know the enum so that it can cast with `m_filterBy->currentData().value<TransferListModel::Column>()`
otherwise i get an error at compile like `Type is not registered, please use the Q_DECLARE_METATYPE macro to make it known to Qt's meta-object system`